Extensions/messaging_tweaks.js

```diff
@@ -366,6 +372,9 @@ XKit.extensions.messaging_tweaks = new Object({
 		if (XKit.extensions.messaging_tweaks.preferences.hide_send_post.value) {
 			XKit.tools.add_css(".messaging-share-post-search, .messaging-share-post-main {display:none;}", "messaging_tweaks");
 		}
+		if (XKit.extensions.messaging_tweaks.preferences.hide_online_indicators.value) {
+			$(".inbox-row.status-indicator-wrapper").hide();
```

Comment:
This doesn't work. Should be a space between `.inbox-row` and `.status-indicator-wrapper` I think.

![indicator](https://user-images.githubusercontent.com/13437011/29464419-76ba3f18-83fb-11e7-819a-334cd40bee43.png)